### PR TITLE
OKTA-578000 Add Authenticator key, type, constraint and method relationship section

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add Authenticator key, type, constraint and method relationship table
- **Is this PR related to a Monolith release?** No

### Preview:
https://64188f3017f67507e7490226--reverent-murdock-829d24.netlify.app/docs/reference/api/policy/#authenticator-key-type-method-and-characteristic-relationships-for-constraints

image

image

### Resolves:

* [OKTA-578000](https://oktainc.atlassian.net/browse/OKTA-578000)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/e6f231d9611c905766167652c2178f5bb2d74bfe/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 72 | 370 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4a46f5ac-d5d8-430f-b153-fbc7ff882534) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.761faff2-7b53-42a0-b315-452e127b62d8?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/194b11ebfd1fbb240116823fdda1520299a3a021/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 72 | 371 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6ba10faf-fe1e-42a5-96f1-1cc9e092b330) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.8acb7363-98ac-4b13-97c5-144b1cf1bba5?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/1f67bb1a10bd0567f2f48aec16649763c93b963d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 72 | 369 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/95bd6e6c-993e-4f81-9bad-5328d62fe7cc) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.9919c420-d79c-4e85-a18f-16a128ff6205?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/b065db3723899b9ec834d1269a4ce9e1f72d998a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 72 | 366 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fbbe6764-ea93-47c3-bc77-4939b32b4b0c) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.d24fb5f7-759f-4605-9b60-9e58895c9cb9?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 1]
```diff
@@ -2092,13 +2090,43 @@ The number of Authenticator class constraints in each Constraint object must be
...
 | `reauthenticateIn`   | String (ISO 8601) | The duration after which the user must re-authenticate regardless of user activity. This re-authentication interval overrides the [Verification Method object](#verification-method-object)'s `reauthenticateIn` interval.     | ISO 8601 period format for recurring time intervals (for example: `PT1H`) | N/A|
 
+#### Authenticator key, type, method, and characteristic relationships for constraints
+
+The following table shows the possible relationships between all the authenticators, their methods, and method characteristics to construct constraints for a policy.
+
+> **Note**:
+> * Method characteristics with an asterisk (*) indicate that the condition is only satisfied with certain configurations, devices, or flows.
+> * Inherence (user verifying characteristic) constraint can't be configured, but can be satisfied as the second part of a 2FA assurance if the device or platform supports biometrics.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:2106]:
```suggestion
> * You can't configure inherence (user verifying characteristic) constraint, but you can satisfy it as the second part of a 2FA assurance if the device or platform supports biometrics.
```

**Review by susanharper-okta — APPROVED** [review]
one comment - approving to not hold you up

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/165255b26a86bfc13fa7c3d0e6fa47ee613ee6c2/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | [64](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d7ffb387-7074-4b64-bcba-5070050c400b) | 62  | 77  | 27  | 84  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.0fab7645-7111-47db-a48c-252c346f0376?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/2e151bd2b625c7fabb6e16fd47ad10a655477fba/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | [64](https://okta.acrolinx.cloud/api/v1/checking/scorecards/28fb66c6-c0aa-45b0-9733-cad3c506e693) | 62  | 78  | 27  | 84  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.2e76c734-d85f-42c5-8403-376b87596fd2?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**